main.py

```diff
@@ -15,21 +18,21 @@
 old_coins = ["CHESS","OTHERCRAP"]
 
 # loads local configuration
-config = load_config('config.yml')
+config = load_config(settings.config_folder + '/config.yml')
```

Comment:
I think it makes sense to use os.path.join() instead of the + operator.
I will look cleaner and there is no need for slashes at the beginning of the filename